Makefile

```diff
@@ -466,35 +466,37 @@ deb_version:
 out/minikube_$(DEB_VERSION).deb: out/minikube_$(DEB_VERSION)-0_amd64.deb
 	cp $< $@
 
+out/minikube_$(DEB_VERSION)-0_%.deb: PACKAGING_DIRECTORY=out/minikube_$(DEB_VERSION)-$*
 out/minikube_$(DEB_VERSION)-0_%.deb: out/minikube-linux-%
-	cp -r installers/linux/deb/minikube_deb_template out/minikube_$(DEB_VERSION)
-	chmod 0755 out/minikube_$(DEB_VERSION)/DEBIAN
-	sed -E -i 's/--VERSION--/'$(DEB_VERSION)'/g' out/minikube_$(DEB_VERSION)/DEBIAN/control
-	sed -E -i 's/--ARCH--/'$*'/g' out/minikube_$(DEB_VERSION)/DEBIAN/control
+	cp -r installers/linux/deb/minikube_deb_template $(PACKAGING_DIRECTORY)
+	chmod 0755 $(PACKAGING_DIRECTORY)/DEBIAN
+	sed -E -i 's/--VERSION--/'$(DEB_VERSION)'/g' $(PACKAGING_DIRECTORY)/DEBIAN/control
+	sed -E -i 's/--ARCH--/'$*'/g' $(PACKAGING_DIRECTORY)/DEBIAN/control
 	if [ "$*" = "amd64" ]; then \
-	    sed -E -i 's/--RECOMMENDS--/virtualbox/' out/minikube_$(DEB_VERSION)/DEBIAN/control; \
+	    sed -E -i 's/--RECOMMENDS--/virtualbox/' $(PACKAGING_DIRECTORY)/DEBIAN/control; \
 	else \
-	    sed -E -i '/Recommends: --RECOMMENDS--/d' out/minikube_$(DEB_VERSION)/DEBIAN/control; \
+	    sed -E -i '/Recommends: --RECOMMENDS--/d' $(PACKAGING_DIRECTORY)/DEBIAN/control; \
 	fi
-	mkdir -p out/minikube_$(DEB_VERSION)/usr/bin
-	cp $< out/minikube_$(DEB_VERSION)/usr/bin/minikube
-	fakeroot dpkg-deb --build out/minikube_$(DEB_VERSION) $@
-	rm -rf out/minikube_$(DEB_VERSION)
+	mkdir -p $(PACKAGING_DIRECTORY)/usr/bin
+	cp $< $(PACKAGING_DIRECTORY)/usr/bin/minikube
+	fakeroot dpkg-deb --build $(PACKAGING_DIRECTORY) $@
+	rm -rf $(PACKAGING_DIRECTORY)
 
 rpm_version:
 	@echo $(RPM_VERSION)
 
 out/minikube-$(RPM_VERSION).rpm: out/minikube-$(RPM_VERSION)-0.x86_64.rpm
 	cp $< $@
 
+out/minikube_$(RPM_VERSION)-0_%.deb: PACKAGING_DIRECTORY=out/minikube_$(RPM_VERSION)-$*
```

Comment:
Note that the original said "minikube-$(RPM_VERSION)" (with hyphen, not underscore)

Now we get a race between the deb and the rpm instead, and it still fails to build `make -j`

Reply:
Need to use two different variables:

```
DEB_PACKAGING_DIRECTORY=out/minikube_$(DEB_VERSION)-$*
RPM_PACKAGING_DIRECTORY=out/minikube-$(RPM_VERSION)-$*
```

Next time, we'll use mktemp instead.